wercker.yml

```diff
@@ -0,0 +1,29 @@
+box: ubuntu:15.04
+build:
```

Comment:
this is just for building, its not intended to test the compatibility of the cookbook recipes.  Still working on how we can do test-kitchen.. I'm thinking of setting up test-kitchen-ec2 to do this.